packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -1140,6 +1140,25 @@ void main() {
     expect(inputBox.hitTest(BoxHitTestResult(), position: inputBox.globalToLocal(newFourthPos)), isFalse);
   });
 
+  testWidgets('minlines cannot be larger than maxLines', (WidgetTester tester) async {
+    try {
+      await tester.pumpWidget(
+        overlay(
+          child: Container(
+            width: 300.0,
+            child: SelectableText(
+              'abcd',
+              minLines: 4,
+              maxLines: 3,
+            ),
+          ),
+        ),
+      );
+    } on AssertionError catch (e) {
+      expect(e.toString(), contains('(maxLines == null) || (minLines == null) || (maxLines >'));
```

Comment:
nit: since this is more human-readable
```suggestion
      expect(e.toString(), contains("minLines can't be greater than maxLines"));
```